src/user.cpp

```diff
@@ -121,7 +128,11 @@ OptionalMessage User::ExNickCommand(const Event& event){
 		ret_message = this->nick_name_ + " changed his nickname to " + new_nickname + ".\n";
 	}
 	this->nick_name_ = new_nickname;
-	return OptionalMessage::Create(this->fd_, ret_message);
+	if (IsVerified() && !this->is_displayed_welcome()) {
+		set_displayed_welcome(true);
+		return OptionalMessage::Create(get_fd(), utils::GetWelcomeString());
+	}
+	return OptionalMessage::Create(get_fd(), ret_message);
```

Comment:
NICKは認証前だと何も表示しないようですが、そこは合わせない方針で行きますか？
ちなみに合わせるなら以下の通りです。
```suggestion
	if (IsVerified()) {
		if (!this->is_displayed_welcome()) {
			set_displayed_welcome(true);
			return OptionalMessage::Create(get_fd(), utils::GetWelcomeString());
		}
		return OptionalMessage::Create(get_fd(), ret_message);
	}
	return OptionalMessage::Empty();
```

Reply:
提案ありがとうございます、
反映しましょう